### ISSUE TITLE
NullPointerException while importing WAR file

### ISSUE THREAD

**Issue by thc202** [secdec/attack-surface-detector-zap#15]
Hopefully it's ok to raise the issue here (it does not seem to be a problem in ASD code).

Steps to reproduce the issue:
1. Run ZAP with ASD add-on installed;
2. Import the BodgeIt WAR (https://github.com/psiinon/bodgeit/releases)
3. Note that nothing happens, in the log there's an exception:
```
ERROR org.zaproxy.zap.ZAP$UncaughtExceptionLogger  - Exception in thread "AWT-EventQueue-1"
java.lang.NullPointerException
	at java.io.File.<init>(File.java:277)
	at com.denimgroup.threadfix.framework.impl.jsp.JSPEndpointGenerator.applyLineNumbers(JSPEndpointGenerator.java:299)
	at com.denimgroup.threadfix.framework.impl.jsp.JSPEndpointGenerator.<init>(JSPEndpointGenerator.java:175)
	at com.denimgroup.threadfix.framework.engine.full.EndpointDatabaseFactory.getDatabase(EndpointDatabaseFactory.java:176)
	at com.denimgroup.threadfix.framework.engine.full.EndpointDatabaseFactory.getDatabase(EndpointDatabaseFactory.java:150)
	at com.denimgroup.threadfix.framework.engine.full.EndpointDatabaseFactory.getDatabase(EndpointDatabaseFactory.java:143)
	at com.denimgroup.threadfix.framework.engine.full.EndpointDatabaseFactory.getDatabase(EndpointDatabaseFactory.java:110)
	at com.denimgroup.threadfix.framework.engine.full.EndpointDatabaseFactory.getDatabase(EndpointDatabaseFactory.java:89)
	at org.zaproxy.zap.extension.attacksurfacedetector.AttackSurfaceDetectorPanel.getEndpoints(AttackSurfaceDetectorPanel.java:372)
	at org.zaproxy.zap.extension.attacksurfacedetector.AttackSurfaceDetectorPanel.access$200(AttackSurfaceDetectorPanel.java:76)
	at org.zaproxy.zap.extension.attacksurfacedetector.AttackSurfaceDetectorPanel$1.actionPerformed(AttackSurfaceDetectorPanel.java:157)
[...]
```

Versions:
ASD v1.1.1
ZAP 2.7.0
Java 1.8.0_171

**Comment by matthewD-AVI** [comment]
It is fine to raise this issue here and we are working on addressing it right now.

**Comment by tylercamp** [comment]
I've discovered the issue in the ASTAM Correlator HAM dependency, which is specific to UNIX-based hosts. I'm currently testing the fix and will deploy an update to Maven within a few hours.

**Comment by tylercamp** [comment]
The fixed modules have been pushed to Maven as 1.2.14.

**Comment by matthewD-AVI** [comment]
New module is being implemented in the ASD plugins and has been encapsulated in try catch blocks resulting in an error message if something does go wrong. Resolved as of commit 3ea068d2e4483ab0e476c08084a5447f034ece87

**Comment by thc202** [comment]
Thanks!
